setup.py

```diff
@@ -38,7 +38,7 @@
     classifiers=CLASSIFIERS,
     setup_requires=["setuptools_scm"],
     install_requires=[
-        "Appium-Python-Client",
+        "Appium-Python-Client<3",
```

Comment:
I added v3 support here #29 . No need this restriction anymore unless there is something else that doesn't work anymore. At least release notes didn't indicate any problems..